presto-resource-group-managers/src/test/java/com/facebook/presto/resourceGroups/db/TestResourceGroupsDao.java

```diff
@@ -117,14 +117,12 @@ private static void testSelectorUpdate(H2ResourceGroupsDao dao, Map<Long, Select
                 2,
                 Optional.of(Pattern.compile("ping.*")),
                 Optional.of(Pattern.compile("ping_source")));
-        map.remove(2);
```

Comment:
Notice I had to read the commit again to invent the message, while the message should have helped me to understand the commit and its validity at the first reading